keps/sig-node/3008-qos-class-resources/README.md

```diff
@@ -891,6 +925,39 @@ available within each of these resource types.
 +}
 ```
 
+#### Consts
+
+We define standard well-known QoS-class resource types in the API. These are
+considered as the canonical list of "official" QoS-class resource names. In
+addition to the name the API documents the expected high-level behavior and
+semantics of these canonical QoS-class resources to provide common standards
+across different implementations.
+
+<<[UNRESOLVED @sftim]>>
+The canonical Kubernetes names for QoS-class resources are non-namespaced (i.e.
+without a `<namespace>/` prefix). Namespaced (or fully qualified) names like
+`example.com/acme-qos` are not controlled and are meant for e.g. vendor or
+application specific QoS implementations.
+<<[/UNRESOLVED]>>
+
+```diff
++const (
++       // QoSResourceRdt is the name of the QoS-class resource named IntelRDT
++       // in the OCI runtime spec and interfaced through the resctrlfs
++       // pseudp-filesystem in Linux. This is a container-level reosurce.
++       QoSResourceIntelRdt = "rdt"
```

Comment:
"QoS" is going to confuse coder that tries to extract other_style_names from BumpyCapsWithAcronyms.

QoSResource -> qo_s_resource

I propose QOS in names.

(another example this done badly is IPv6 -> I_Pv6)

Reply:
Naming is hard :) -> Changed as suggested